app/src/main/java/com/example/easysurvey/ui/theme/Theme.kt

```diff
@@ -17,14 +17,7 @@ private val LightColorPalette = lightColors(
         primaryVariant = intermediatePurple,
         secondary = white
 
-        /* Other default colors to override
-    background = Color.White,
-    surface = Color.White,
-    onPrimary = Color.White,
-    onSecondary = Color.Black,
```

Comment:
Comments were not being used and therefore increasing technical debt, cited from https://wakeupandcode.com/todo-comments-and-technical-debt/
https://kotlinlang.org/docs/coding-conventions.html#source-code-organization